docs/designers-developers/developers/block-api/block-registration.md

```diff
@@ -204,11 +204,22 @@ example: {
                 'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Praesent et eros eu felis.'
             ),
         },
-    },
+	},
```

Comment:
should we indent the whole object here using tabs? (otherwise, we might want to keep this for a separate PR and revert this line)

Reply:
Done